ouroboros-consensus-byron/src/Ouroboros/Consensus/Byron/Ledger/Block.hs

```diff
@@ -68,10 +69,13 @@ import           Ouroboros.Consensus.Byron.Ledger.Orphans ()
 -------------------------------------------------------------------------------}
 
 newtype ByronHash = ByronHash { unByronHash :: CC.HeaderHash }
-  deriving stock   (Eq, Ord, Show, Generic)
+  deriving stock   (Eq, Ord, Generic)
   deriving newtype (ToCBOR, FromCBOR, Condense)
   deriving anyclass (NoUnexpectedThunks)
 
+instance Show ByronHash where
+  showsPrec = Quiet.qshowsPrec
```

Comment:
My preference is to use deriving via everywhere for `Quiet`. We already embrace deriving via in consensus, so I see no reason to not use it (or do you?)

Reply:
Only reason: I favor the more simple/explicit/established/etc things when the trade-off is minimal.

But I'm not arguing. So I'll push a commit to use DerivingVia throughout.